packages/keystone/package.json

```
@@ -12,6 +12,7 @@
     "./preprocessors": "./preprocessors/index.js",
     "./rateLimiting": "./rateLimiting/index.js",
     "./sentry": "./sentry/index.js",
+    "./fileAdapter/": "./fileAdapter/",
```

This notation should point on the concrete executable file)
```suggestion
    "./fileAdapter/": "./fileAdapter/file Adapter.js",
```

---

There is two options. You can export every executable that you need to use outside packages module. Or you can export only `index.js` file with reexport of all of the modules. That's up to you)